text/0092-replay-issue-creation.md

```diff
@@ -0,0 +1,55 @@
+- Start Date: 2023-05-15
+- RFC Type: decision
+- RFC PR: https://github.com/getsentry/rfcs/pull/92
+- RFC Status: draft
+
+# Summary
+
+We want to detect certain categories of issues only available through the Session Replay product. The first of these issues is the "Slow Click" detection issue.
+
+# Motivation
+
+1. Inform the long-term strategy for detecting and ingesting Session Replay generated issues.
+2. Provide actionable feedback to developers that could not otherwise be captured in a performance span or error event.
+3. Increase product awareness in free-tier customers and encourage adoption.
+
+# Options Considered
```

Comment:
I'm trying to think of a way to make sure we're looking at an exhaustive list of options, even if some are bad. 

here's where i'm starting from:

| SDK | Network data | Relay & Ingest | Result |
| --- | --- | --- | --- |
| @sentry/javascript | error payload (call `captureException()` in browser)| normal processing | error quota risk |
| @sentry/javascript | click detection (same payload as in experiment) | server-side detector needed | - | 
| @sentry/replay | existing breadcrumbs + rrweb events | normal relay processing | server-side detector needed | replay must be sampled |
| @sentry/replay | click detection breadcrumb (same payload as in experiment) | server-side detector needed | replay must be sampled |

Reply:
@ryan953 @cmanallen Could we add a column here that is "Product Experience" or "Product Implications" so we can visualize what this means to our users?